python/tank/context.py

```diff
@@ -984,7 +1061,7 @@ def context_yaml_constructor(loader, node):
 ################################################################################################
 # utility methods
 
-def _task_from_sg(tk, task_id):
+def _task_from_sg(tk, task_id, additional_fields = None):
```

Comment:
see answer above - what do you think?  Change it and accept executing the hook twice or leave it?